course-v2/layouts/partials/course_info.html

```diff
@@ -8,15 +8,19 @@
         Course Info
       </div>
       {{ if $inPanel }}
-      <div
-        class="font-black close-course-info large-and-above-only"
-        id="{{ if $isDesktopCourseDrawer }}hide-desktop-course-drawer{{ end }}">
+      <button
+        type="button"
+        class="btn close-course-info large-and-above-only"
+        id="{{ if $isDesktopCourseDrawer }}hide-desktop-course-drawer{{ end }}"
+        aria-label="Close Course Info"
+        >
         <img
           class="toggle navbar-toggle offcanvas-toggle"
           src="/images/close_small.svg"
           alt=""
+          width="12px"
```

Comment:
`close_small.svg` was a bit bigger than `left_arrow.svg` hence looked inconsistent when closing/opening the drawer so reducing its size here to minimize this inconsistency